update-dependencies/Invoke-UpdateDependencies.ps1

```diff
@@ -13,13 +13,13 @@ $ErrorActionPreference = 'Stop'
 $imageName = "update-dependencies"
 
 try {
-    & docker build -t $imageName -f $PSScriptRoot\Dockerfile --pull $PSScriptRoot
+    & docker build -t $imageName -f $PSScriptRoot\Dockerfile --pull $PSScriptRoot\..
```

Comment:
Minor - replace this `$PSScriptRoot\..` with `$repoRoot` assuming the declaration from line 21 is moved up,

Reply:
Good suggestion - fixed.